imports/plugins/core/system-info/server/no-meteor/queries/systemInformation.js

```diff
@@ -0,0 +1,16 @@
+import packageJson from "/package.json";
+
+/**
+ * @name queries.systemInformation
+ * @method
+ * @memberof SystemInformation/GraphQL
+ * @summary get systemInformations
+ * @param {Object} context - an object containing the per-request state
+ * @return {<Object} System Information
+ **/
+export default async function systemInformation(context) {
+  return {
+    apiVersion: packageJson.version,
+    plugins: await context.collections.Packages.find({ version: { $exists: true } }).toArray()
```

Comment:
Hmm, just thinking this through a bit more. There are `settings` objects on many package docs that may have sensitive settings. Although this won't make it through GraphQL back to clients, I'd rather be safe and filter out unneeded fields here. So add a `.map` on the array and keep only `name` and `version` properties for now.

Another thing I forgot is that there is one Package doc per shop ID in a multi-shop scenario. So the GraphQL will need to take a `shopId` argument and pass it into this function, and then this function should include it in the `find` query.

Finally, `version` could be `null`, which passes `{ $exists: true }` check, so `{ $ne: null }` is better (ref: https://docs.mongodb.com/manual/tutorial/query-for-null-fields/).

So putting all that together, something like this:

```js
export default async function systemInformation(context, { shopId }) {
  const plugins = await context.collections.Packages.find({ shopId, version: { $ne: null } }).toArray();
  return {
    apiVersion: packageJson.version,
    plugins: plugins.map(({ name, version }) => ({ name, version }))
  };
}
```